untitled/app/controllers/Application.java

```diff
@@ -0,0 +1,14 @@
+package controllers;
+
+import play.*;
+import play.mvc.*;
+
+import views.html.*;
+
+public class Application extends Controller {
```

Comment:
Not sure...... I think it was like that when I opened the file. What's it on yours?